### ISSUE TITLE
Edit Post

### ISSUE THREAD

**Issue by michitheonlyone** [michitheonlyone/refbook-laravel#5]
Add the ability to edit/update a post. The post should only be editted by the creator.

**Comment by ajaxStardust** [comment]
I thought it was already capable of editing/ updating a post. 
[depth_ping_debug_01-2024-03-12_20.17.43.webm](https://github.com/michitheonlyone/refbook-laravel/assets/94220040/56c3fa60-91d7-4b63-a2cc-9ce837d6f0aa)

Are you identifying a need to have the editing of post #A1 only editible if User #B2 is the original author of post #A1? 

P.S. It's Jeff from LinkedIn 
:)

**Comment by michitheonlyone** [comment]
Yes it has edit already build in but it needs more control of who can edit. The best would be to use some kind of RBAC or permission system where an admin can edit all posts and a user can edit only their own. I leave it open on how to make it more flexible "multiuser"-ish. Show me your creativity and ingenuity 😉
